src/common/MuxConfig.h

```diff
@@ -120,6 +120,22 @@ class MuxConfig
     */
     inline void setNegativeStateChangeRetryCount(uint32_t stateChangeRetryCount) {
         mNegativeStateChangeRetryCount = stateChangeRetryCount;
+        // this is to ensure for the same amount of link prober self and peer
+        // events, self link prober state change could be triggered first
+        setPeerNegativeStateChangeRetryCount(2 * stateChangeRetryCount);
```

Comment:
Is doubling necessary? Link prober times out based on its own clock, theoretically simply +1 should give it enough time window to trigger self unknown event first? 

I'm just thinking we should still take the failover performance into consideration, in case peer do need help on switching to standby.

Reply:
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/35479537/195742693-78325fd3-8a76-42c9-acd4-9b3d4d276816.png)
For the above scenario, simply +1 could not resolve the issue.